README.md

```diff
@@ -209,6 +209,21 @@ user and blocks access to key chia locations.
 user with limited permissions.
 * For running in docker, you can check out @ajacobson repository for
 [chiadog-docker](https://github.com/ajacobson/chiadog-docker).
+
+## Filters
+
+Filters are also supported by adding them under the `filter` section, i.e.
+
+```yaml
+filter:
+  transaction_amount: "0.000000000002"
+```
+
+The above example will filter out all chia messages for transactions that are not bigger than `0.000000000002`. This is
+useful in reducing spam that is often a result of chia 
```

Comment:
Thanks for documenting this. We typically document anything config related directly inside the [config-example.yaml](https://github.com/martomi/chiadog/blob/main/config-example.yaml). Please add the new configs there and feel free to create a multiline comment describing the purpose and function of the setting.

The benefit is that users can ideally understand all possible configs by only looking at the example config and not cross-referencing other parts of the docs.